Certificates/cert-gen.py

```diff
@@ -0,0 +1,103 @@
+# Requires inkscape in path and PDF module
+import argparse
+import csv
+import numpy as np
+import os
+import pathlib
+from PDF import PdfFileReader, PdfFileWriter
+from subprocess import run
+
+def argument_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'template_svg',
+        help="Input svg template with text matching 'NAME OF STUDENT IN CAPS' and 'NAME OF SCHOOL IN CAPS' for "
+             "replacement with the students' names and schools'/teams' names, respectively, "
+             "on the produced certificates.",
+        type=pathlib.Path,
+    )
+    parser.add_argument(
+        '-i', '--input_data',
+        help="Input csv with columns of 'First Name', 'Last Name', and 'School'."
+             "In absence of this argument, the script's own test data will be used.",
+        nargs=1,
+        type=pathlib.Path,
+    )
+    parser.add_argument(
+        '-o','--output_path',
+        help="Path in which the output will be deposited. Defaults to current working directory.",
+        type=pathlib.Path,
+        nargs=1,
+        default=os.getcwd(),
+    )
+    return parser
+
+def main():
+    options = argument_parser().parse_args()
+    
+    outpath = pathlib.Path(options.output_path, "Certificates.pdf")
+    tempsvgpath = pathlib.Path(options.output_path, ".temp-cert.svg")
+    temppdfpath = pathlib.Path(options.output_path, ".temp-cert.pdf")
+    
+    if options.input_data == None:
+        firstnames = ["", "WWWWWWW",
+                      "Cave", "Aleph",
+                      "Theresa", "Blon Fel-Fotch Passameer-Day"]
+        lastnames = ["", "WWWWWWWW",
+                     "Johnson", "Null",
+                     "Green", "Slitheen"]
+        schools = ["", "WWWWWWWWWWWWWWWWWWWWWWWWW",
+                   "Aperture Fixtures", "Hilbert's Hotel",
+                   "L33T Haxxor$ and C00L K@s", "High Raxicoricofallapatorian School of Interplanetary Conquest"]
+    else:
+        with open(str(options.input_data)) as datacsv:
+            csvreader = csv.reader(datacsv)
+            rows = np.array([row for row in csvreader])
+        headings = list(rows[0])
+        data = rows[1:]
+        firstnames = data[:, headings.index("First Name")]
+        lastnames = data[:, headings.index("Last Name")]
+        schools = data[:, headings.index("School")]
+            
+    studentno = len(firstnames)
+    
+    print(options.template_svg)
+    with options.template_svg.open() as tempfile:
+        template = tempfile.read()
```

Comment:
You might enjoy using `pathlib.Path`'s [`.read_text()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text) method, which would reduce these two lines to one: `template = options.template_svg.read_text()`.